tests/test_feature_detection.py

```diff
@@ -0,0 +1,41 @@
+"""Unit test related to feature detection model instanciation
+"""
+
+from keras.models import Model
+
+from deeposlandia.keras_feature_detection import FeatureDetectionNetwork
+
+def test_simple_network_architecture():
+    """Test the instanciation of a simple feature detection network
```

Comment:
Could you remove the word `instanciation`?

Reply:
Done.